packages/flutter_tools/lib/src/web/web_constants.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const String kWasmMoreInfo = 'See https://flutter.dev/wasm for more information.';
```

Comment:
i think we already have a file with constant strings, fwiw.

Reply:
We have `UserMessages` but there's talk of removing it - https://github.com/flutter/flutter/issues/125155

I don't see any other good candidates (I did look...)